content/00.build/30.zksync-sso/50.getting-started.md

```diff
@@ -0,0 +1,106 @@
+---
+title: Getting Started
+description: Learn how to integrate ZKsync SSO into your app.
+---
+
+Integrating ZKsync SSO into your app requires minimal amount of changes.
+::callout{icon="i-heroicons-light-bulb"}
+ZKsync SSO can be used standalone or added as one of the options in the onboarding library of your choice.
+::
+
+## Quick Start
+
+Just 2 steps:
+
+1. Install the ZKsync SSO SDK package.
+
+  ::code-group
+
+    ```bash [npm]
+    npm i zksync-account
```

Comment:
The final package name will be `zksync-sso` right?

Reply:
Yes, updated